lxd/container_lxc.go

```diff
@@ -1921,6 +1921,12 @@ func (c *containerLXC) initLXCIPVLAN(cc *lxc.Container, networkKeyPrefix string,
 	}
 
 	if m["ipv4.address"] != "" {
+		//Check necessary sysctls are configured for use with l2proxy parent in IPVLAN l3s mode.
+		sysctlVal, err := networkSysctlGet(fmt.Sprintf("ipv4/conf/%s/forwarding", m["parent"]))
```

Comment:
As mentioned in my comment, you are not in fact checking `err` here, You should either check for it separately or if you don't care about potential errors, then don't assign err.

Reply:
@stgraber agreed, thanks, what tool is that you are running to detect those?